packages/manager/modules/pci/src/projects/onboarding/onboarding.routing.js

```diff
@@ -18,7 +21,56 @@ export default /* @ngInject */ ($stateProvider) => {
             : null,
         ),
     resolve: {
-      breadcrumb: () => null, // Hide breadcrumb
+      goToCreateNewProject: /* @ngInject */ ($state) => () =>
+        $state.go('pci.projects.new'),
+
+      goToCreateDiscoveryProject: /* @ngInject */ ($state) => () =>
+        $state.go('pci.projects.discovery'),
+
+      model: /* @ngInject */ (pciProjectNew, me) =>
+        pciProjectNew.createOrderCart(me.ovhSubsidiary).then((cart) => ({
+          agreements: false,
+          description: 'discoveryProject',
+          cart,
+          hds: cart?.hdsItem !== undefined,
+        })),
+
+      cart: /* @ngInject */ (me, pciProjectNew) =>
+        pciProjectNew
+          .createOrderCart(me.ovhSubsidiary)
+          .then((newOrderCart) =>
+            pciProjectNew.getOrderCart(me.ovhSubsidiary, newOrderCart.cartId),
+          ),
+
+      setCartProjectItem: /* @ngInject */ (model, cart, pciProjectNew) => () =>
+        pciProjectNew.setCartProjectItemDescription(cart, model.description),
+
+      summary: /* @ngInject */ (getSummary) =>
+        getSummary().then((summary) => summary),
```

Comment:
```suggestion
        getSummary(),
```
So you might merge `summary` and `getSummary` resolve objects